src/robot_dart/control/manifold_control.hpp

```diff
@@ -0,0 +1,25 @@
+#include <robot_dart/control/pd_control.hpp>
+
+namespace robot_dart {
+    namespace control {
+
+        class ManifoldControl: public PDControl {
```

Comment:
The name (`Manifold`) suggests a different type of controller. I would just enhance the existing `PDControl` class.

Reply:
Agreed